### PR TITLE
Me function

### LINKED ISSUE #11: Add /me feature to the bot.
## First-time Contributors

**If it is the first time that you contribute to the bot, follow these steps:**

1. You need to have **git & golang** available on your machine. 
    - To install Git - https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
    - To setup golang, follow the doc here - https://golang.org/doc/install
2. Write a comment in this issue thread to let other possible contributors know that you are working on this bug. For eg : `Hey all, I would like to work on this issue.`
3. Ping [Botfather](https://t.me/BotFather) on Telegram and make your instance of OSDC-Bot bot by selecting `/newbot` from the options it provides.
4. Copy the `TELEGRAM_TOKEN` provided by Botfather. 
5. Fork this repo.
6. Run `git clone https://github.com/<YOUR_USERNAME>/bots.git && cd bots/telegram-bot`
7. If you have installed golang, run `go build .` 
8. Wait ⏳
9. Run `export TELEGRAM_TOKEN=<botfather_token>`
10. Now, run `./telegram-bot`. The bot would be running at the user handle provided by you.
11. Make changes according to the issue. Test the working of the changes.
12. Commit the changes and then run `git push origin master` and then open a PR! 🎉

### PR DESCRIPTION
**Pull request by PrimalPimmy**
Fixes #11 .

Introducing a new feature in the OSDC telegram-bot , a /me function.  You can do role-playing just like how it works in IRC!
Here's a GIF of the feature working. 

me

### CODE REVIEW

File: telegram-bot/main.go [hunk 1]
```diff
@@ -56,6 +57,11 @@ func tweet() {
 
 }
 
+func me(user string, ID int64, msgtext string) {
+
+	bot.Send(tbot.NewMessage(ID, "*@"+user+" "+msgtext+"*"))
```
Comment (daemon1024) [inline telegram-bot/main.go]:
```suggestion
	bot.Send(tbot.NewMessage(ID, "* @"+user+" "+msgtext+" *"))
```

File: telegram-bot/main.go [hunk 2]
```diff
@@ -252,6 +258,14 @@ func main() {
...
 			case "paste":
 				paste(ID, update.Message)
+			case "me":
+				user := update.Message.From.UserName
+				msg := update.Message.Text
+				s := strings.SplitN(msg, " ", 2)
+				if len(s) == 2 {
+					me(user, ID, s[1])
+					bot.DeleteMessage(tbot.NewDeleteMessage(ID, update.Message.MessageID))
+				}
```
Comment (daemon1024) [inline telegram-bot/main.go]: The entire code should be abstracted to the `me` function instead of just `bot.Send`

**Review by daemon1024 — CHANGES_REQUESTED** [review]

File: telegram-bot/main.go (hunk 2 above)
Comment (PrimalPimmy) [reply telegram-bot/main.go]: I couldn't put the bot delete command inside the function, I don't think the user update text would be defined inside the function, if there is a way, do tell. I have committed the changes of what I could do.

File: telegram-bot/main.go [hunk 3]
```diff
@@ -252,6 +261,9 @@ func main() {
 				fetchnote(ID, update.Message.Text, *client)
 			case "paste":
 				paste(ID, update.Message)
+			case "me":
+				me(update.Message.From.UserName, ID, update.Message.Text)
```
Comment (daemon1024) [inline telegram-bot/main.go]: Just pass the whole update.Message then each of it's methods would be available to you.

```suggestion
				me(update.Message, ID)
```

File: telegram-bot/main.go [hunk 4]
```diff
@@ -56,6 +57,14 @@ func tweet() {
 
 }
 
+func me(user string, ID int64, msgtext string) {
```
Comment (daemon1024) [inline telegram-bot/main.go]:
```suggestion
func me(message *tbot.Message, ID int64) {
```

File: telegram-bot/main.go (hunk 2 above)
Comment (daemon1024) [reply telegram-bot/main.go]: Pass the whole message interface and you should be able to access all the underlying functions.

Comment (PrimalPimmy) [reply telegram-bot/main.go]: It is done, marking as resolved.

File: telegram-bot/main.go [hunk 5]
```diff
@@ -56,6 +57,15 @@ func tweet() {
 
 }
 
+func me(message *tbot.Message, ID int64) {
```
Comment (daemon1024) [inline telegram-bot/main.go:60]: Can you add some comments explaining what this function is about? Rest LGTM 👍

**Review by daemon1024 — APPROVED** [review]
LGTM :partying_face:

**Pull request merged**